public/styleguide.html

```diff
@@ -2,7 +2,7 @@
ird-level .nav-item,nav.top-level .nav-item{cursor:pointer;height:100%;display:flex;align-items:center}nav.second-level ul,nav.third-level ul,nav.top-level ul{display:flex;align-items:center}nav.second-level>.container,nav.third-level>.container,nav.top-level>.container{grid-column-start:2;grid-column-end:12;height:100%}nav.second-level>.container>.title,nav.third-level>.container>.title{width:180px;word-break:break-word}nav.second-level ul,nav.third-level ul,nav.top-level ul{height:100%}nav.top-level{height:40px;color:#fff;color:var(--color-white);background-color:#39546a;background-color:var(--color-primary)}nav.top-level .title{padding-right:30px}nav.top-level .nav-item{padding:10px 20px}nav.top-level .nav-item.active,nav.top-level .nav-item:focus,nav.top-level .nav-item:hover{background-color:#4b7293;background-color:var(--color-primary-g1)}nav.top-level .nav-item{padding:0 20px;color:#fff;color:var(--color-white);font-weight:200;font-size:14px}nav.second-level{background-color:#d7e8ed;background-color:var(--color-accent-g2);height:80px;box-shadow:0 2px 3px -1px #d6d6d6;box-shadow:0 2px 3px -1px var(--color-alto)}nav.second-level .nav-item,nav.third-level .nav-item{padding:0 25px}nav.third-level .nav-item{margin:0 25px}nav.second-level .nav-item{position:relative}nav.second-level .nav-item:after{content:" ";width:0;height:0;border-left:10px solid transparent;border-right:10px solid transparent;border-bottom:10px solid transparent;position:absolute;left:calc(50% - 16px);bottom:-3px;z-index:3;transition:border-bottom-color .2s ease-in}nav.second-level .nav-item.active,nav.second-level .nav-item:focus,nav.second-level .nav-item:hover{background-color:transparent;color:#39546a;color:var(--color-primary)}nav.second-level .nav-item.active:after,nav.second-level .nav-item:focus:after,nav.second-level .nav-item:hover:after{border-bottom-color:#fff;border-bottom-color:var(--color-white)}nav.third-level{height:60px}nav.third-level h3{margin:0;text-transform:capitalize;font-weight:600;color:#39546a;color:var(--color-primary);padding-bottom:10px;font-size:22px}@media (max-width:1400px){nav.third-level h3{font-size:18px;padding-bottom:5px}nav.third-level .nav-item{font-size:14px;padding:10px}}@media (max-width:1490px){nav.second-level>.container,nav.third-level>.container,nav.top-level>.container{grid-column-start:1;margin-left:50px;margin-right:50px}}nav.third-level{padding-top:5px}nav.third-level .container{border-bottom:1px solid #d7e8ed;border-bottom:1px solid var(--color-accent-g2)}nav.third-level .nav-item{font-size:16px;letter-spacing:.02em;border-bottom:3px solid transparent}nav.third-level .nav-item.active,nav.third-level .nav-item:focus,nav.third-level .nav-item:hover{text-decoration:none;color:#39546a;color:var(--color-primary);border-bottom-color:#39546a;border-bottom-color:var(--color-primary);transition:border-bottom-color .2s ease-in,color .2s ease-in}.psg-main nav.second-level,.psg-main nav.third-level,.psg-main nav.top-level{width:1000px}
```

Comment:
Not sure if this is okay or a problem - I think it happened when I ran the `yarn` task after making the other changes...

Reply:
Looks ok.